src/hooks/loadables/useLoadSpendingLimits.ts

```diff
@@ -76,11 +75,11 @@ export const useLoadSpendingLimits = (): AsyncResult<SpendingLimitState[]> => {
   const chainId = useChainId()
   const provider = useWeb3ReadOnly()
 
-  // Update spending limits whenever a transaction is executed
+  // Update spending limits whenever the tx history updates
   // TODO: Find a more optimised way to update them
```

Comment:
I think with the change this TODO is done.

Reply:
Thanks, I removed the comment